cmd/backup-manager/app/restore/manager.go

```diff
@@ -17,16 +17,20 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
+	"os/exec"
+	"path"
 	"strconv"
+	"strings"
 	"time"
 
+	"github.com/pingcap/errors"
 	"github.com/pingcap/tidb-operator/cmd/backup-manager/app/constants"
-	"github.com/pingcap/tidb-operator/cmd/backup-manager/app/util"
+	backupUtil "github.com/pingcap/tidb-operator/cmd/backup-manager/app/util"
```

Comment:
```suggestion
	backuputil "github.com/pingcap/tidb-operator/cmd/backup-manager/app/util"
```
The package alias convention is using lowercase.